MiniSim/MiniSim.swift

```diff
@@ -157,29 +162,50 @@ class MiniSim: NSObject {
                     UNUserNotificationCenter.showNotification(title: "Derived data has been cleared!", body: "Removed \(amountCleared) of data")
                     NotificationCenter.default.post(name: .commandDidSucceed, object: nil)
                 }
-            default:
-                break
             }
         }
     }
     
-    private func populateSections() {
-        if !menu.items.isEmpty {
-            return
+    private var devicesListHeaders: [NSMenuItem] {
+        var sections: [DeviceListSection] = []
+        if UserDefaults.standard.enableiOSSimulators {
+            sections.append(.iOS)
         }
-        MenuSections.allCases.forEach { item in
-            if (!item.attachItem) {
-                return
-            }
-            
-            let menuItem = item.menuItem
-            if menuItem.tag >= MenuSections.clearDerrivedData.rawValue {
-                menuItem.action = #selector(menuItemAction)
-                menuItem.target = self
+        
+        if UserDefaults.standard.enableAndroidEmulators {
+            sections.append(.android)
+        }
+    
+        guard sections.count > 1 else {
```

Comment:
We should check if `sections.count > 0` because in that case using just android or just iOS simulators doesn't work

Reply:
Oh, right, good catch. Updated.